library/classes/InsuranceNumbers.class.php

```diff
@@ -51,8 +51,8 @@ function populate()
     function insurance_numbers_factory($provider_id)
     {
         $ins = array();
-        $sql = "SELECT id FROM "  . $this->_table . " where provider_id = '" . $provider_id . "' order by insurance_company_id";
-        $results = sqlQ($sql);
+        $sql = "SELECT id FROM "  . $this->_table . " WHERE provider_id = ? ORDER BY insurance_company_id";
```

Comment:
escape the table name via the escape_table_name() function
(and also do for the queries like this below)